server.go

```diff
@@ -862,15 +862,15 @@ func (sp *serverPeer) OnGetAddr(_ *peer.Peer, msg *wire.MsgGetAddr) {
 	// Do not accept getaddr requests from outbound peers.  This reduces
 	// fingerprinting attacks.
 	if !sp.Inbound() {
-		peerLog.Debugf("Ignoring getaddr request from outbound peer ",
+		peerLog.Debugf("Ignoring getaddr request from outbound peer "+
 			"%v", sp)
```

Comment:
Shouldnt this be

peerLog.Debugf("Ignoring getaddr request from outbound peer %v", sp)

Reply:
Yes, but then the line is too long, so we break it.